source/features/clean-conversations.css

```diff
@@ -1,8 +1,14 @@
-/* Remove user avatar from new comment form on threads */
-.review-thread-reply .avatar {
+/* Remove user avatar from new comment form */
+:is(.review-thread-reply, #issuecomment-new) .avatar {
 	display: none;
 }
 
 .review-thread-reply .review-thread-reply-button {
 	margin-left: 0;
 }
+
+/* Hide arrow on new comment form */
+.timeline-comment--caret::before,
```

Comment:
Does this really only  target the form? The selector is generic

Reply:
Good catch, this was done in a hurry in my previous lint PR 😅